cdk/apps/ica_credentials/app.py

```diff
@@ -36,6 +36,24 @@
     ),
 )
 
+# Staging
+IcaCredentialsDeployment(
+    app,
+    f"{CDK_APP_NAME}-stg",
+    "c9173925-a838-4394-9fc6-61cb93c252a1",
+    [
+        # staging_workflows
+        # FIXME
```

Comment:
@reisingerf Might also need to create `staging_workflows` project? @alexiswl Could you pls fill/update for FIXME here, please.

Reply:
Flo; also need fill master API key for Staging, pls. Please ditto README and, respective sec name in SM Console. Thanks.

Reply:
> Might also need to create staging_workflows project?
@victorskl no, workflows from both the development_workflows and production_workflows are available in the staging project

Reply:
> In a sense, no tailored token needed for workflows to be appeared in Staging project context. So, empty list here would just work, I reckon?

Nope, the workflows token is used by the cwl-ica GitHub actions but that's it

Reply:
Ah. I need help explain, pls. 😅 Shall we have a quick call, pls.

Reply:
As discussed. Empty list here is ok for now; as Staging workflows are pointed from either DEV or PROD.